var/spack/repos/builtin/packages/hpgmg/package.py

```diff
@@ -59,13 +59,11 @@ def configure_args(self):
             args.append('--no-fv')
         else:
             # Apple's Clang doesn't support OpenMP
-            if not (self.spec.satisfies('%clang') and self.compiler.is_apple):
+            if not self.spec.satisfies('%apple-clang'):
                 cflags.append(self.compiler.openmp_flag)
```

Comment:
This may no longer be valid since I added support for `self.compiler.openmp_flag` to `apple-clang`

Reply:
This is an improvement on the package, which is fine but I would do it in another PR.